src/main/java/com/synopsys/integration/detect/ApplicationUpdater.java

```diff
@@ -284,6 +307,35 @@ private void checkEnvironmentProperties() {
             proxyPropertiesPut(ARG_PROXY_USERNAME, utility.getSysEnvProperty(SYS_ENV_PROP_PROXY_HTTP_USERNAME));
             proxyPropertiesPut(ARG_PROXY_PASSWORD, utility.getSysEnvProperty(SYS_ENV_PROP_PROXY_HTTP_PASSWORD));
         }
+        
+        if (!springbootEnvMap.isEmpty()) {
+            updateFromSpringBootPropertiesIfAny();
```

Comment:
Spring Boot Application system property has higher rank than System properties. Here, the System properties if any are overridden by any available Spring Boot Application properties.